x-pack/plugins/index_management/__jest__/client_integration/index_template_wizard/template_create.test.tsx

```diff
@@ -558,7 +558,6 @@ describe('<TemplateCreate />', () => {
             name: TEMPLATE_NAME,
             indexPatterns: DEFAULT_INDEX_PATTERNS,
             allowAutoCreate: true,
-            dataStream: {},
```

Comment:
Should we update the way the `completeStepOne` helper works? Looks like we are passing `dataStream: {}`, which would suggest we want to test configuring with a data stream. However, now that the default behavior is to have it toggled on, this [line](https://github.com/elastic/kibana/blob/main/x-pack/plugins/index_management/__jest__/client_integration/index_template_wizard/template_form.helpers.ts#L177) behaves differently.

Reply:
Nice catch! I've now updated the completeStepOne to deal with this in a better way